packages/python-packages/apiview-copilot/evals/deconstruct_testcase.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+import json
+import pathlib
+
+
+if __name__ == "__main__":
```

Comment:
Seems like these could easily be plugged into the `cli.py` script at the root of the workspace folder. under an `eval` group.

Reply:
I was able to plug in the create / deconstruct test case into the CLI, but the evals script will still need to be run by `python evals/run.py` for now (there is some issue with the evals library needing to be called from __main__)